ips/services/new_run_steps/step_5.py

```diff
@@ -1,81 +1,104 @@
 from flask import request, render_template, session, jsonify
 from ips.util.ui_logging import log
-
+from flask_wtf.file import FileRequired
 
 from ips.services import app_methods
 from ips.services.forms import LoadDataForm
 
-
 def run_step_5(run_id):
-    form = LoadDataForm()
+    from remote_pdb import RemotePdb
+    #RemotePdb('0.0.0.0', 4445).set_trace() 
     error = False
-
     run_info = app_methods.get_run(run_id)
-    if form.validate_on_submit():
-
+        
+    #RemotePdb('0.0.0.0', 4445).set_trace() 
+    
+    # Build validator dictionary based on values in run_info
+    meta={
+        'survey_file': run_info['SURVEY_FILE'],
+        'shift_file': run_info['SHIFT_FILE'],
+        'non_response_file': run_info['NR_FILE'],
+        'unsampled_file': run_info['UNSAMPLED_FILE'],
+        'tunnel_file': run_info['TUNNEL_FILE'],
+        'sea_file': run_info['SEA_FILE'],
+        'air_file': run_info['AIR_FILE']
+    }
+    form = LoadDataForm(meta=meta)
+    #RemotePdb('0.0.0.0', 4445).set_trace() 
+
+    if form.validate():
```

Comment:
I don't think this is right? Not sure why you've changed from `validate_on_submit()` to `validate()` - let's discuss.